src/App.tsx

```diff
@@ -12,212 +118,93 @@ export const App: React.FC = () => {
     if (newTodoField.current) {
       newTodoField.current.focus();
     }
+
+    if (user) {
+      getTodos(user.id)
+        .then(setTodos)
+        .catch(() => setIsError('Something went wrong...'));
+    }
   }, []);
 
+  const filteredTodos = useMemo(() => {
+    return todos.filter(todo => {
+      switch (filteredStatus) {
+        case FilterStatus.COMPLETED: {
+          return todo.completed;
+        }
+
+        case FilterStatus.ACTIVE: {
+          return !todo.completed;
+        }
+
+        default:
+          return todo;
+      }
+    });
```

Comment:
Don't filter todos in default case or if filter status all
```suggestion
      switch (filteredStatus) {
        case FilterStatus.COMPLETED: {
          return todos.filter(todo => (
            todo.completed;
          )

        case FilterStatus.ACTIVE: {
          .........
          
        case FilterStatus.ALL:
        default:
          return todos;
      }
    });
```